CDCgov/NEDSS-Modernization
# [CNFT2-1024] - Save Page As Template

## Description

[CNFT2-1024] - Save Page As Template

## Tickets

* [Jira Ticket] https://cdc-nbs.atlassian.net/browse/CNFT2-1024

## Steps to verify

Run the CURL command in postman
```
curl --location 'http://localhost:8095/nbs/page-builder/api/v1/template/save' \
--header 'Authorization: bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnb3YuY2RjLm5icyIsImlhdCI6MTY5OTQ3MDQxOCwiZXhwIjoxNjk5NDcyMjE4LCJzdWIiOiJzdXBlcnVzZXIifQ.26k6w0N8LNXKDJ8gm892T_k3ZalwBGgfR--YGSGvUps' \
--header 'Content-Type: application/json' \
--data '{
    "waTemplateUid": 1000333,
    "templateName": "TestTemplate",
    "templateDescription": "Test Template API"
}'
```
Verify the response in dbo.WA_template table

[CNFT2-1024]: https://cdc-nbs.atlassian.net/browse/CNFT2-1024?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
